### ISSUE TITLE
Sort Directory DA: Something broken in Tip-of-Tree?

### ISSUE THREAD

**Issue by inexorabletash** [inexorabletash/a2d#119]
The Sort Directory DA triggers "A window must be closed before opening this directory", then graphics state is trashed.

**Comment by inexorabletash** [comment]
Now I can't reproduce it. :(

**Comment by inexorabletash** [comment]
Sort Directory does something odd on exit; it pushes a new return value onto the stack. ($400B = `JT_SELECT_WINDOW`-1); altering the calling stack could have borked this...

**Comment by inexorabletash** [comment]
And a direct call to JT_SELECT_WINDOW causes a hang. Bummer...

There's also weird stack manipulation. Seems flaky.

**Comment by inexorabletash** [comment]
Sort Directory is hitting JT_SELECT_WINDOW with A (window_id) of $FF ! Not good...

**Comment by inexorabletash** [comment]
Somehow FrontWindow's result value (storing to $A) is getting clobbered...

**Comment by inexorabletash** [comment]
"This should never have worked".

* When DeskTop was launching a DA it configured MGTK ZP usage differently than normal
* The Sort Directory DA attempted to use a ZP location to retrieve the top window
* MGTK would smash the ZP
* The top window would be bogus when calling JT_SELECT_WINDOW, resulting in shenanigans. 

I dropped the reconfiguration of MGTK's use of the ZP... which makes the problem go away and the directory refresh work. I don't think this is something I broke, but the problem is flaky, depending on MGTK internal state (it seems)
